owi.opam.template

```diff
@@ -1 +1,4 @@
 available: (arch = "x86_32" | arch = "x86_64" | arch = "arm64") & os != "win32"
+depexts: [
+  ["llvm" "clang" "lld" "wabt"]  {os-family = "debian"}
```

Comment:
```suggestion
  ["llvm" "clang" "lld" "wabt"]  {os-family = "debian"}
  ["llvm" "wabt"]  {os-family = "osx"}
```
CI is failing, and it appears to be because we're not specifying the dependencies here. However, I'm not quite sure if these are the only packages required for macOS. Additionally, I couldn't find mentions of `lld` or `clang` in Homebrew packages, so I assume they must be included in `llvm`.

Reply:
Thanks. I added this line directly myself as running `dune b owi.opam` was needed to generate the new `owi.opam` file...

Let's see if this is enough :sweat_smile:

Reply:
This appears to have done nothing :sweat_smile: 

I was looking at the `os-family` available in [here](https://github.com/ocaml-opam/opam-depext#opam-depext-plugin) and perhaps wrongfully assumed `osx` would work. Inspecting the CI, it looks like it's looking for `os-family = "homebrew"`? I have no current way to test this tho, sorry.

```sh
...
# Detecting depexts using vars: arch=x86_64, os=macos, os-distribution=homebrew, os-family=homebrew
...
```

Reply:
Well, still not. Let's merge as the CI is green on Linux. We'll fix macOS later.

Reply:
At least it's installing `wabt` and `llvm` not. But it's still saying `opt: command not found`

Reply:
Yes I saw... @krtab is going to have a look.